FWIO/RNTuple/plugins/RNTupleSource.cc

```diff
@@ -227,10 +232,26 @@ namespace edm {
 
   void RNTupleSource::fillDescriptions(ConfigurationDescriptions& descriptions) {
     ParameterSetDescription desc;
-    desc.addUntracked<std::string>("fileName");
+    desc.addUntracked<std::vector<std::string>>("fileNames")->setComment("only the first file will be used for now");
     desc.addUntracked<bool>("enableMetrics", false);
     desc.addUntracked<bool>("useClusterCache", true);
 
+    //make interface compatible with PoolSource. Some will be marked obsolete once available
```

Comment:
Is the `Some will be marked obsolete once available` part of the comment still accurate?

Reply:
That part is wrong as I did switch to obsolete for those that we will never use. I used optional for ones that we should make work in the future.

Reply:
done